datafusion/expr/src/udf.rs

```diff
@@ -809,6 +837,10 @@ impl ScalarUDFImpl for AliasedScalarUDFImpl {
         self.inner.output_ordering(inputs)
     }
 
+    fn output_preserves_lex_ordering(&self, inputs: &[ExprProperties]) -> Result<bool> {
```

Comment:
Seems like you don't need the `output_` prefix, just `preserves_lex_ordering` is enough.

Reply:
Resolved in 91c7012.